CodeGeneration/Sources/generate-swiftsyntax/templates/swiftsyntaxbuilder/BuildableNodesFile.swift

```diff
@@ -94,13 +91,15 @@ private func createConvenienceInitializer(node: Node) throws -> InitializerDeclS
     return nil
   }
 
-  let params = ParameterClauseSyntax {
-    FunctionParameterSyntax("leadingTrivia: Trivia? = nil", for: .functionParameters)
-    for param in normalParameters + builderParameters {
-      param
+  let params = ParameterClauseSyntax(
+    parameterList: FunctionParameterListSyntax {
```

Comment:
Do you know why we can’t use the result builder syntax like before anymore and now need to write `parameterList: FunctionParameterListSyntax`? Are we missing a convenience initializer now?

Reply:
My fault. Sorry